src/openzaak/conf/includes/base.py

```diff
@@ -221,6 +222,14 @@
     "openzaak.config.bootstrap.demo.DemoUserStep",
 ]
 
+# Note that by default UTC times are used here (see `nowfun` kwarg)
+CELERY_BEAT_SCHEDULE = {
+    "daily-remove-imports": {
+        "task": "openzaak.import_data.tasks.remove_imports",
+        "schedule": crontab(hour="9"),
+    }
+}
```

Comment:
Open Zaak doesn't have celery beat yet, so the required configuration to set that up will have to be added as well

Reply:
This is added through https://github.com/open-zaak/open-zaak/pull/1693/commits/387901e7728bda8403af42f0d5cf975d95632bb0. Although `django-celery-beat` [is not mandatory for using periodic tasks](https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html#using-custom-scheduler-classes) (like the one which is being added through this PR) I think it is the most practical way to add it. The alternative is to use the default scheduler (which removes the need for adding `django-celery-beat`) which saves the schedule in a file on the filesystem (which could be removed when a docker setup is used and not configured with any persistent storage solution for it).